int16 bumper1
int16 bumper2
int16 bumper3
int16 bumper4
int16 bumper5
int16 bumper6
int16 bumper7
int16 bumper8

